src/components/AccountForms/RegistrationForm/index.js

```diff
@@ -204,20 +202,40 @@ const RegistrationForm = ({ mockOnSubmit }) => {
     retypePassword: false,
   });
   const history = useHistory();
+  // const [error, setError] = useState({ userName: { message: '' } });
+
+  /*const key = Object.keys(error || {}).toString();
+  if (key === 'email') {
+    const { email } = error;
+    errors.email = email;
+  }*/
 
   const onSubmit = (data) => {
-    request.post(API_PATH.REGISTER, data).then(({ status }) => {
-      if (status === 200) {
-        const inProd = process.env.NODE_ENV === 'production';
-        const options = {
-          category: 'onSubmit',
-          action: 'Created an Account',
-        };
-        inProd && event(options);
-        dispatch(updateAuth(auth.checkCookie()));
-        history.push('/mentors');
-      }
-    });
+    request
```

Comment:
i see axios is installed, why not axios here?

Reply:
we're using an interceptor